diff -p -up mapserver-5.2.1/mapogcsld.c~ mapserver-5.2.1/mapogcsld.c
--- mapserver-5.2.1/mapogcsld.c~	2008-11-17 14:55:37.000000000 +0100
+++ mapserver-5.2.1/mapogcsld.c	2009-01-09 10:28:56.000000000 +0100
@@ -4955,8 +4955,8 @@
         {
             if (i > 0 && i < nElements-1)
             {
-                sprintf(szAttribute, aszElements[i-1]);
-                sprintf(szValue, aszElements[i+1]);
+                sprintf(szAttribute, "%s", aszElements[i-1]);
+                sprintf(szValue, "%s", aszElements[i+1]);
 
                 /* parse attribute */
                 nLength = strlen(szAttribute);
@@ -4990,7 +4990,7 @@ char *msSLDParseExpression(char *pszExpr
                     else if (szValue[0] == '\"')
                        bDoublequote = 1;
                     else
-                      sprintf(szFinalValue,szValue);
+                      sprintf(szFinalValue, "%s", szValue);
                     
                     iVal = 0;
                     if (bSinglequote || bDoublequote)
diff -p -up mapserver-5.2.1/mapows.c~ mapserver-5.2.1/mapows.c
--- mapserver-5.2.1/mapows.c~	2008-06-20 14:31:29.000000000 +0200
+++ mapserver-5.2.1/mapows.c	2009-01-09 10:22:08.000000000 +0100
@@ -1598,7 +1598,7 @@ char *msOWSGetProjURN(projectionObj *pro
         else if( strcasecmp(tokens[i],"imageCRS") == 0 )
             sprintf( urn, "urn:ogc:def:crs:OGC::imageCRS" );
         else if( strncmp(tokens[i],"urn:ogc:def:crs:",16) == 0 )
-            sprintf( urn, tokens[i] );
+            sprintf( urn, "%s", tokens[i] );
         else
             strcpy( urn, "" );
 
